### PR TITLE
[test_chassis] Fix test_chassis on API tests

### PR DESCRIPTION
**Pull request by shlomibitton**
Signed-off-by: Shlomi Bitton <shlomibi@nvidia.com>

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
Each platform support different LED colors, but the functionality is the same.
This fix group LED colors into groups of same logical meaning and test accordingly, with no need to define vendor and platform specific LED colors.

#### How did you do it?
Group same logical LED colors into different Led "types".

#### How did you verify/test it?
Run the test on any platform.

### CODE REVIEW

**Comment by shlomibitton** [comment]
retest vsimage please

File: tests/platform_tests/api/test_chassis.py [hunk 1]
```diff
@@ -364,22 +365,49 @@ def test_sfps(self, duthost, localhost, platform_api_conn):
...
         # TODO: Get a platform-specific list of available colors for the status LED
-        LED_COLOR_LIST = [
-            "off",
-            "red",
+
+        FAULT_LED_COLOR_LIST = [
             "amber",
-            "green",
+            "red",
+            "orange"
```
Comment (jleveque) [inline tests/platform_tests/api/test_chassis.py:372]: Why do we need both `amber` and `orange`? They are essentially the same color.

**Comment by shlomibitton** [comment]
retest vsimage  please

**Comment by shlomibitton** [comment]
retest vsimage please

File: tests/platform_tests/api/test_chassis.py (hunk 1 above)
Comment (shlomibitton) [reply tests/platform_tests/api/test_chassis.py]: Fixed.

**Comment by liat-grozovik** [comment]
@wangxin  can you please review?

**Review by wangxin — APPROVED** [review]

**Pull request merged**